proto/meta.proto

```diff
@@ -259,6 +259,7 @@ message Relation {
     catalog.Index index = 4;
     catalog.View view = 5;
     catalog.Function function = 6;
+    catalog.Connection connection = 7;
```

Comment:
Connections should be also included in `MetaSnapshot`, so that all frontends can initialize connection catalogs in frontend from them.